a3p-integration/proposals/z:acceptance/test/stakeBld.test.js

```diff
@@ -10,6 +10,8 @@ import assert from 'node:assert';
 import process from 'node:process';
 import { networkConfig, agdWalletUtils } from './test-lib/index.js';
 
+const BLD_COIN_DENOM = 'ubld';
+const BLD_DELEGATION = 10;
```

Comment:
I don't see any advantage for making `BLD_DELEGATION` a Number instead of a BigInt.


```suggestion
const BLD_DELEGATION = 10n;
```